python/cudf/setup.py

```diff
@@ -48,6 +48,14 @@
 
 
 class build_ext_and_proto(build_ext):
+    def build_extensions(self):
+        try:
+            # Silence the '-Wstrict-prototypes' warning
+            self.compiler.compiler_so.remove("-Wstrict-prototypes")
+        except Exception:
+            pass
```

Comment:
`self.compiler.compiler_so.remove()` isn't a long-running call, it only removes the flag from the list of flags. The try/catch is here in case accessing `self.compiler.compiler_so` throws an error.

Reply:
I mixed them up anyway, `Exception` is the one that doesn't include keyboard interrupts and whatnot which is what we want. Resolving.